sympy/combinatorics/fp_groups.py

```diff
@@ -1338,13 +1347,14 @@ def elimination_technique_2(C):
     C._schreier_generators = gens
     return C._schreier_generators, C._reidemeister_relators
 
-def reidemeister_presentation(fp_grp, H, C=None ,rel=False):
+def reidemeister_presentation(fp_grp, H, C=None ,homomorphism=False):
     """
     fp_group: A finitely presented group, an instance of FpGroup
     H: A subgroup whose presentation is to be found, given as a list
     of words in generators of `fp_grp`
-    rel: When set to True, this computes the elements in the `fp_grp`
-         to which the shreier generators corresponf to.
+    homomorphism: When set to True, this computes the elements in the `fp_grp`
+                  to which the shreier generators correspond to and returns a
+                  homomorphism between the presentation and the parent group.
```

Comment:
Same comment as for `define_schreier_generators` - in fact the same docstring would work if we are returning a dictionary.